### PR TITLE
WIP: Sec3bench

### PR DESCRIPTION
**Pull request by IainNZ**

### CODE REVIEW

**Comment by mlubin** [comment]
I'd like to do a single PR for the whole revision, so I'll push to this branch

**Comment by IainNZ** [comment]
Sure

**Comment by IainNZ** [comment]
Need to update the main README Julia instructions I guess too

File: acpower/opf_662bus.gms [hunk 1]
```diff
@@ -232,7 +232,8 @@ obj_def.. obj =e= sum(bus_k$(bus_type(bus_k) ne 0), sqr(bus_p_load(bus_k) +
 
 
 MODEL acpower /all/;
+acpower.holdfixed = 1;
```
Comment (mlubin) [inline acpower/opf_662bus.gms:242]: Maybe add a comment explaining the origin and impact of this option?

**Comment by IainNZ** [comment]
Added a comment about holdfixed

**Comment by mlubin** [comment]
Updated the readme. We also need to update the notebooks, especially for the NLP changes.

**Comment by IainNZ** [comment]
I'll do that now

**Comment by IainNZ** [comment]
Notebooks updated

**Pull request merged**